server/app.js

```diff
@@ -9,39 +9,44 @@ import { createServer } from 'http';
 import { Model } from 'objection';
 import swaggerDocument from '@/swagger.json';
 import Routes from '@routes/v1';
+import dotenv from 'dotenv';
```

Comment:
We dont want to have this. that's why we have the config. please remove this and have a look at the config files.

Reply:
please which of the config file

Reply:
done